### ISSUE TITLE
Loco app not allowing permission to take screenshots.

### ISSUE THREAD

**Issue by awadhesh204** [dipendrapkrl/TriviaHelper#44]
Hi, after the recent update of the Loco app they have disabled the permissions to take screenshots and hence Trivia Helper is not able to take images for OCR. Would there be any workaround this? All other live quizzing apps might switch to this tactic and then Trivia Helper would become useless.

**Comment by TonyStark** [comment]
hmm, then it need root permission to capture screenshot i think...

**Comment by awadhesh204** [comment]
I don't think that's possible either on Nougat. Also, I really don't want to root my phone and install xposed modules. Can you guide me on how to decompile and recompile an app so that I can disable the FLAG_SECURITY inside the app itself?

**Comment by TonyStark** [comment]
maybe app cloner app can do that, but i dont know really

**Comment by meuserhere** [comment]
reinstall the previous version again

**Comment by awadhesh204** [comment]
I've done that already but even this would be a temporary solution as they can make this version obsolete after a few more updates.

**Comment by meuserhere** [comment]
You are right, it gonna be expired soon after they update it to some higher security level. Fact is that other than a knowledgeable brain, tricks are temporary. Day by day they update their apps to higher security, our tricky masters will come with someting new. Don't be hopelss brother

**Comment by awadhesh204** [comment]
You're absolutely right, there's no substitute for that. But most of the people, like me, use this app only for a backup because you just can't win one single game based on this app entirely. I've a few cases where it gives a definite wrong answer. Since it's a paid app and I've paid for it, I assumed that the developer would have had some workaround this issue of screenshot.

**Comment by meuserhere** [comment]
This app is google based. As per question, it matches the options and counts them from its searching page, accordingly shows the result. So it completely depends on google. 
Take an example Q. Who was the first person to hit triple century in test cricket? 
You google it. Very rare, almost no entry for that correct person 'Andy Sandham'. You wouldn't get that correct name anywhere easily other than Virendar Sehwag, Karun nair etc. In that case, app result will be Virender sehwag by huge margin probably.
A developer can't change the whole google. 
Trivia team first check their questions by passing them into this type of hacking app, then they deliver 2/3 of such type google confusing queations in each game.

**Comment by akashchaurasiya** [comment]
Kindly, give us solution for loco and swoo.
